utility/proto/message.proto

```diff
@@ -4,156 +4,51 @@ package utility;
 option go_package = "github.com/pokt-network/pocket/utility/types";
 
 import "vote.proto";
-import "session.proto";
 import "google/protobuf/any.proto";
-import "google/protobuf/timestamp.proto";
+
+enum ActorType {
+  App = 0;
+  Node = 1;
+  Fish = 2;
+  Val = 3;
+}
 
 message MessageSend {
   bytes from_address = 1;
   bytes to_address = 2;
   string amount = 3;
+  ActorType actor_type = 4;
 }
 
-message MessageStakeServiceNode {
+message MessageStake {
   bytes public_key = 1;
   repeated string chains = 2;
   string amount = 3;
-  string service_url = 4;
+  optional string service_url = 4;
```

Comment:
Optional was removed in proto3, but seems like it's an experimental feature again according to https://github.com/protocolbuffers/protobuf/blob/main/docs/implementing_proto3_presence.md.

If we're adding this back, can you pass `--experimental_allow_proto3_optional` in the Makefile? I think it's a noop otherwise

![Screen Shot 2022-07-25 at 12 16 26 PM](https://user-images.githubusercontent.com/1892194/180856918-665ae7a8-c2d0-4dd1-b94c-4428da5f13cb.png)